examples/plot_anomaly_bench.py

```diff
@@ -0,0 +1,118 @@
+"""
+==================================
+Anomaly detection's ROC benchmarks
+==================================
+
+This example benchmarks outlier detection algorithms using ROC curves.
```

Comment:
This example benchmarks outlier detection algorithms using ROC curves on classical anomaly detection datasets. The algorithm performance is assessed in an outlier detection context: